aziotctl/aziotctl-common/src/config/apply.rs

```diff
@@ -117,7 +117,7 @@ pub fn run(
                                         &identity_cert.method
                                     {
                                         set_est_auth(
-                                            auth.as_ref(),
+                                            &auth,
```

Comment:
Only revert the `str::split_once` part of this commit. The rest is okay.

Reply:
Fixed. Had to also remove some clippy changes.